generators/pkg/writer/write.go

```diff
@@ -22,17 +21,34 @@ func WriteToDisk(filename string, writer io.Writer) error {
 	}
 	defer file.Close()
 
-	spec, err := parser.Parse(file)
+	github, err := parser.Parse(file)
 	if err != nil {
 		return fmt.Errorf("unable to parse the file from %s: %w", filename, err)
 	}
 
-	task := generator.GenerateTask(spec)
+	task := generator.GenerateTask(github)
+	pipeline, err := generator.GeneratePipeline(github)
+
+	// write task
+	if _, err = writer.Write([]byte("---\n")); err != nil {
```

Comment:
```suggestion
	if _, err := writer.Write([]byte("---\n")); err != nil {
```

here and instances below as well.